scene/gui/graph_node.cpp

```diff
@@ -668,15 +668,15 @@ void GraphNode::_port_pos_update() {
 		if (slot_table.has(slot_index)) {
 			if (slot_table[slot_index].enable_left) {
 				PortCache port_cache;
-				port_cache.pos = Point2i(edgeofs, vertical_ofs + size.height / 2);
+				port_cache.pos = Vector2(port_cache.pos).rotated(get_rotation());
```

Comment:
Meant that part specially, not the first part, so:
```suggestion
				port_cache.pos = Point2(edgeofs, vertical_ofs + size.height / 2).rotated(get_rotation());
```